config.rb

```diff
@@ -1,25 +1,8 @@
+#Compass Sass configuration file
```

Comment:
Será que não daria para movermos estas configurações para o `config.ru`? Dessa forma, só precisamos manter um arquivo de configuração para o projeto. Acredito que daria pra abrir uma outra PR para alterar isso depois que esta aqui for mergeada. O que acham?

Pelo que parece, as pessoas que criaram o Sinatra recomendam o uso do Sprockets: 
http://recipes.sinatrarb.com/p/asset_management/sprockets#article